tools/monitoring/config.json

```
@@ -0,0 +1,49 @@
+{
+  "database": {
+    "directory": "waspdb"
+  },
+  "logger": {
+    "level": "debug",
```

```suggestion
    "level": "info",
```
maybe let's set `info` as the default? debug is too spammy for normal operations, I think

---

btw, why not use the default `config.json` that is on the root of the project?

---

The config in the root project doesn't have the metrics plugin turned on. I wasn't sure if it's something we'll always want to have turned on for the root config thats why I created this one.

---

okay, makes sense 👍